apps/src/blockly/addons/workspaceSvgFrame.js

```diff
@@ -106,7 +106,6 @@ function onWorkspaceChange(event) {
   if (
     [
       Blockly.Events.DELETE,
-      Blockly.Events.MOVE,
```

Comment:
should this change be here?

Reply:
Oh, good catch! I tested this locally, but didn't mean to commit it.